brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyInteractor.swift

```diff
@@ -95,14 +95,9 @@ class BuyInteractor: NSObject, Interactor, BuyViewActions {
             case .success(let quote):
                 self?.dataStore?.quote = quote
                 
-                // TODO: Should be updated when BE returns the right value.
-                // (quote?.timestamp ?? 0) + (9 * 60 * 1000) = 1 Min + 540.000 Seconds = 10 Mins.
-                self?.dataStore?.expirationTimestamp = (quote?.timestamp ?? 0) + (9 * 60 * 1000)
-                
-                self?.presenter?.presentExchangeRate(actionResponse: .init(from: from,
-                                                                           to: toCurrency,
-                                                                           rate: quote?.exchangeRate,
-                                                                           expires: self?.dataStore?.expirationTimestamp))
```

Comment:
revert this part of the code please. i have changes on my repo. we still need self?.dataStore?.expirationTimestamp.